tests/test_tag_object.py

```diff
@@ -30,6 +30,40 @@ def test_tag_object(self):
         )
         self.assertEqual(1, 1)
 
+    def test_tag_object_max_depth(self):
+        payload = {
+            "hello": "world",
+            "level1": {
+                "level2_dict": {"level3": 3},
+                "level2_list": [None, True, "nice", {"l3": "v3"}],
+                "level2_bool": True,
+                "level2_int": 2,
+            },
+            "vals": [{"thingOne": 1}, {"thingTwo": 2}],
+        }
```

Comment:
Shouldn't we add a value with a deeper level to see if the it doesn't appear in the `function.request`?

Reply:
So the deeper-level items will be converted into one string (so they still appear but in a big chunk of string). For example, the "lv3", "level3" etc are already the "deeper level" of the setting here (2), and they would end up in the string value....

Or.... do you mean add an lv4 item?

Reply:
Ah I see, thank you so much, I was misunderstanding the feature!